wmf2svg/src/net/arnx/wmf2svg/Main.java

```diff
@@ -22,7 +22,7 @@
 import java.lang.reflect.Proxy;
 import java.util.logging.Logger;
 import java.util.zip.GZIPOutputStream;
-
+import com.google.typography.font.sfntly.FontFactory;
```

Comment:
I wish this library don't have dependencies for other libraries. If it must have, you need to realize by using a dynamic class loading and  a reflection.

Reply:
No it does not have any dependency

Reply:
A FontFactory class is not a starndard jdk class. So it will add a depedency.